.github/workflows/smoke-checks.yml

```diff
@@ -104,21 +110,25 @@ jobs:
   build-and-test-e2e-debug:
     name: Test E2E UI (Debug)
     runs-on: macos-12
+    strategy:
+      matrix:
+        batch: [0, 1, 2]
```

Comment:
What does `[0, 1, 2]` refer to?

Reply:
The array count stands for the matrix size and the number itself stands for the test batch (i.e.: which portion of the tests will be taken for which matrix job). From now on we are able to extend this array (e.g.: `[0, 1, 2, 3, 4, 5]`) thereby parallelizing our tests even more.